pkg/ingester/http_admin.go

```diff
@@ -53,31 +57,41 @@ const tpl = `
 	</body>
 </html>`
 
-var tmpl *template.Template
+var UserStatsTmpl *template.Template
 
 func init() {
-	tmpl = template.Must(template.New("webpage").Parse(tpl))
+	UserStatsTmpl = template.Must(template.New("webpage").Parse(tpl))
 }
 
-type userStatsByTimeseries []UserIDStats
+type UserStatsByTimeseries []UserIDStats
 
-func (s userStatsByTimeseries) Len() int      { return len(s) }
-func (s userStatsByTimeseries) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+func (s UserStatsByTimeseries) Len() int      { return len(s) }
+func (s UserStatsByTimeseries) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 
-func (s userStatsByTimeseries) Less(i, j int) bool {
+func (s UserStatsByTimeseries) Less(i, j int) bool {
 	return s[i].NumSeries > s[j].NumSeries ||
 		(s[i].NumSeries == s[j].NumSeries && s[i].UserID < s[j].UserID)
 }
 
-// AllUserStatsHandler shows stats for all users.
-func (d *Distributor) AllUserStatsHandler(w http.ResponseWriter, r *http.Request) {
-	stats, err := d.AllUserStats(r.Context())
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
+// UserIDStats models ingestion statistics for one user, including the user ID
+type UserIDStats struct {
+	UserID string `json:"userID"`
+	UserStats
+}
+
+// UserStats models ingestion statistics for one user.
+type UserStats struct {
+	IngestionRate     float64 `json:"ingestionRate"`
+	NumSeries         uint64  `json:"numSeries"`
+	APIIngestionRate  float64 `json:"APIIngestionRate"`
+	RuleIngestionRate float64 `json:"RuleIngestionRate"`
+	ActiveSeries      uint64  `json:"activeSeries"`
+	LoadBlocks        uint64  `json:"loadBlocks"`
```

Comment:
This is unclear to me. I see it just uses `db.Blocks()`. Does it include head block as well?
How we gonna use this data to help with scale down. Do we check `LoadBlocks==0`?

Reply:
This should be "loadedBlocks" ?